src/python/ensureconda/installer.py

```diff
@@ -104,9 +104,16 @@ def exe_suffix() -> str:
 
 
 @contextlib.contextmanager
-def new_executable(target_filename: "StrPath") -> Iterator[IO[bytes]]:
+def new_executable(target_filename: "Path") -> Iterator[IO[bytes]]:
```

Comment:
what is the expected type of `target_filename` here?

Reply:
its a `pathlib.Path`....